index.html

```diff
@@ -27,7 +27,7 @@
                     <li class="nav-link"><a href="#" class="link">About</a></li>
```

Comment:
- Kindly as this is your portfolio, it should have a `title` tag like "your name" Portfolio. Or some meaningful for you than just the word Portfolio. In this way you can show a better title on your application 💯